charts/incubator/external-dns/questions.yaml

```diff
@@ -0,0 +1,332 @@
+          schema:
+            type: string
+            required: true
+            default: cloudflare
+            enum:
+              - value: akamai
+                description: akamai
+              - value: alibabacloud
+                description: alibabacloud
+              - value: aws
+                description: aws
+              - value: aws-sd
+                description: aws-sd
+              - value: azure
+                description: azure
+              - value: azure-dns
+                description: azure-dns
+              - value: azure-private-dns
+                description: azure-private-dns
+              - value: bluecat
+                description: bluecat
+              - value: civo
+                description: civo
+              - value: cloudflare
+                description: cloudflare
+              - value: coredns
+                description: coredns
+              - value: designate
+                description: designate
+              - value: digitalocean
+                description: digitalocean
+              - value: dnsimple
+                description: dnsimple
+              - value: dyn
+                description: dyn
+              - value: exoscale
+                description: exoscale
+              - value: gandi
+                description: gandi
+              - value: godaddy
+                description: godaddy
+              - value: google
+                description: google
+              - value: ibmcloud
+                description: ibmcloud
+              - value: infoblox
+                description: infoblox
+              - value: inmemory
+                description: inmemory
+              - value: linode
+                description: linode
+              - value: ns1
+                description: ns1
+              - value: oci
+                description: oci
+              - value: pdns
+                description: pdns
+              - value: pihole
+                description: pihole
+              - value: plural
+                description: plural
+              - value: rcodezero
+                description: rcodezero
+              - value: rdns
+                description: rdns
+              - value: rfc2136
+                description: rfc2136
+              - value: safedns
+                description: safedns
+              - value: scaleway
+                description: scaleway
+              - value: skydns
+                description: skydns
+              - value: tencentcloud
+                description: tencentcloud
+              - value: transip
+                description: transip
+              - value: ultradns
+                description: ultradns
+              - value: vinyldns
+                description: vinyldns
+              - value: vultr
+                description: vultr
+        - variable: sources
+          label: "Sources"
+          schema:
+            type: list
+            default: ["ingress"]
```

Comment:
Ingress only? not service?
```suggestion
            default:
              - ingress
              - service
```

Reply:
It could be both, if it's a list can users select both?

Reply:
yes they can, but should the default be one of them only?

Reply:
Their docs are too vague for me lol

<img width="809" alt="image" src="https://github.com/truecharts/charts/assets/89483932/1b564903-bea2-4f5c-a065-228e69ac0747">

So I'm ok with leaving Ingress as default

Reply:
lets add both and users can remove after :D

Reply:
> Ingress only? not service?

By default you don't want it to source service afaik.
Not 100% sure.